clang/include/clang/Driver/Options.td

```diff
@@ -8265,6 +8265,11 @@ def internal_externc_isystem : Separate<["-"], "internal-externc-isystem">,
            "implicit extern \"C\" semantics; these are assumed to not be "
            "user-provided and are used to model system and standard headers' "
            "paths.">;
+def internal_iframework : Separate<["-"], "internal-iframework">,
```

Comment:
Should we call this `internal-externc-iframework` since it more closely corresponds to `internal-externc-isystem` than `internal-isystem`? Especially its position in the search ordering system -> internal-system -> internal-externc-system